tests/src/3d/testqgs3dutils.cpp

```diff
@@ -166,6 +167,42 @@ void TestQgs3DUtils::testQgsBox3DDistanceTo()
   }
 }
 
+void TestQgs3DUtils::testQgsBox3DScale()
```

Comment:
There's a whole set of tests for QgsBox3d here -- https://github.com/qgis/QGIS/blob/master/tests/src/python/test_qgsbox3d.py . Probably best to move the new test to sit alongside these.

Reply:
Your are right! I will update the python test! Thanks.